### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by ManarOmari**
In this project, I build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+];
+
+document.addEventListener('DOMContentLoaded', () => {
+  todoTasks.forEach((n) => {
+    const output = document.createElement('div');
+    output.classList.add('listitem');
+    output.setAttribute('id', `${n.index}`);
+    output.innerHTML = `<div><input type="checkbox"> ${n.description}</div> <span class="listitemIcon"><i class="fa-solid fa-ellipsis-vertical"></i></span>`;
+  //  mainContainer.appendChild(output);
+  });
```
Comment (Reem-lab) [inline src/index.js:24]: - [ ] As I can see after running the project I can't see any tasks displayed so kindly fix the error so I can see some of the tasks as required so kindly fix it and submit it again 💯💪🏻💥

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @ManarOmari, 👋🏻👋🏻

Good job so far! 💥💥💯💯
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (ManarOmari) [reply src/index.js]: Sure , Now it works fine 
@Reem-lab can you check again?

**Review by Davidosky007 — APPROVED** [review]
Hi @ManarOmari 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Well written PR description 👍 
* No linter errors 👍 
* Descriptive commit messages 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**